Assets/MixedRealityToolkit/_Core/Definitions/Visualization/MixedRealityVisualizationProfile.cs

```diff
@@ -26,17 +27,31 @@ public bool RenderMotionControllers
         }
 
         [SerializeField]
-        [Implements(typeof(IMixedRealityControllerVisualizer), TypeGrouping.ByNamespaceFlat)]
-        [Tooltip("The concrete Controller Visualizer component to use on the rendered controller model.")]
-        private SystemType controllerVisualizationType;
+        [Implements(typeof(IMixedRealityVisualizationSystem), TypeGrouping.ByNamespaceFlat)]
+        [Tooltip("The concrete Visualizer component to use on the rendered controller model.")]
+        private SystemType visualizationManager;
 
         /// <summary>
         /// The concrete Controller Visualizer component to use on the rendered controller model
         /// </summary>
-        public SystemType ControllerVisualizationType
+        public SystemType VisualizationManager
         {
-            get { return controllerVisualizationType; }
-            private set { controllerVisualizationType = value; }
+            get { return visualizationManager; }
+            private set { visualizationManager = value; }
+        }
+
+        [SerializeField]
+        [Implements(typeof(IMixedRealityVisualizer), TypeGrouping.ByNamespaceFlat)]
+        [Tooltip("The concrete Visualizer component to use on the rendered controller model.")]
+        private SystemType visualizerType;
```

Comment:
Why did you remove this from the profile, where it belongs?

Reply:
It was renamed, not removed.

Instead of ControllerVisualizationType ( The scene visualiser to use) , you have now:
* VisualizationManager - The manager to use
* VisualizerType - The scene visualiser to use